_includes/provider-ocp-add-container.md

```diff
@@ -73,10 +62,11 @@
     5.  Click **Validate** to confirm that {{ site.data.product.title_short }} can connect
         to the OpenShift Container Platform provider.
 
-9.  For the **Prometheus** alerts service, add the Prometheus alerts
-    endpoint in the **Alerts** tab:
+7.  In the **Alerts** endpoint tab optionally configure the alerts service:
 
-    1.  Select a **Security Protocol** method to specify how to
+    1.  Select a service type, or leave disabled.  Prometheus alerts are only supported on OCP v3
```

Comment:
OCPv4 doesn't have the Prometheus Alert Buffer service installed, I've opened an issue to improve the error message if someone tries to enable this on an OCPv4 (https://github.com/ManageIQ/manageiq-providers-openshift/issues/208) but we should at least document this.